examples/zkp_debugger/src/main.rs

```diff
@@ -28,12 +25,12 @@ fn main() {
     let inputs: Vec<ZkpProgramInput> = vec![
         BPField::from(1).into(),
         BPField::from(2).into(),
-        BPField::from(3).into(),
+        BPField::from(4).into(),
```

Comment:
Whichever line is problematic should have a comment saying so, and explain it is for the purpose of debugging.